### PR TITLE
Update rest-client to 0.12.15

### PR DESCRIPTION
**Pull request by liujoshua**

### CODE REVIEW

File: android-sdk/src/main/assets/logback.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<configuration>
```
Comment (liujoshua) [inline android-sdk/src/main/assets/logback.xml:1]: Need to test whether these configs are getting picked up

File: android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeApplication.java [hunk 2]
```diff
@@ -12,6 +14,6 @@
   public void onCreate() {
     super.onCreate();
     JodaTimeAndroid.init(this);
+    //BasicLogcatConfigurator.configureDefaultContext();
```
Comment (liujoshua) [inline android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeApplication.java:17]: Remove if unneeded

File: researchstack-sdk/build.gradle [hunk 3]
```diff
@@ -56,8 +56,9 @@ dependencies {
   compile 'com.madgag.spongycastle:prov:1.54.0.0'
   compile 'com.madgag.spongycastle:pkix:1.54.0.0'
   testCompile 'junit:junit:4.12'
-  testCompile 'org.slf4j:slf4j-simple:1.7.21'
+//  testCompile 'org.slf4j:slf4j-simple:1.7.21'
```
Comment (liujoshua) [inline researchstack-sdk/build.gradle:59]: Remove if not needed

**Review by liujoshua — COMMENTED** [review]
Reviewing my own code because I wrote it before my trip

File: android-sdk/build.gradle [hunk 4]
```diff
@@ -30,10 +37,14 @@ dependencies {
 
   compile 'net.danlew:android.joda:2.9.4.2'
   compile 'com.android.support:appcompat-v7:24.2.1'
```
Comment (dephillipsmichael) [inline android-sdk/build.gradle:39]: AppCompat is at 25 now.  For the appcompat, cardview, preference, design, etc would the best approach to stay relevant be the "+" sign at the end?  

Also, I was wondering if you knew the answer to this question, if i use this library with appcompat v24, but my app.gradle uses v25, will v25 get compiled in or v24?

**Pull request merged**